docs/src/use/configure/configuration-files.md

```diff
@@ -208,6 +208,7 @@ Filenames starting with a dot, such as `.gitignore`, are considered to have only
 
 #### Globally ignoring files with `ignores`
 
+If you want to globally ignores some directory/files, and apply this directive to all configuration objects of the array,  `global ignores` is exactly made for this scenario.
 If `ignores` is used without any other keys in the configuration object, then the patterns act as global ignores. Here's an example:
```

Comment:
```suggestion
If `ignores` is used without any other keys in the configuration object, then the patterns act as global ignores. This way it gets apply to every configuration object. While patterns in non-global `ignores` only match the files or files within directories, patterns in global `ignores` can also match directories. Here's an example:
```

Let's try to match this with above description a bit.

Reply:
I made a big-rewrite of this section in https://github.com/eslint/eslint/pull/19297/commits/c6efc5197988afd344f6e54d788c8a4c7c4d1f48, check that